source/Cosmos.System2/Network/IPv4/IPPacket.cs

```diff
@@ -1,14 +1,26 @@
-﻿using Cosmos.HAL.Network;
+﻿using sys = System;
```

Comment:
Not sure about if this addition can break anything.
would love to hear your thoughts.